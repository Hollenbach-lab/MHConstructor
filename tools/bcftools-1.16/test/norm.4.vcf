##fileformat=VCFv4.2
##contig=<ID=1,length=249250621>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	10	.	T	C	.	.	.	GT	0/1
1	10	.	T	TAC	.	.	.	GT	0/0
1	10	.	T	TATAC	.	.	.	GT	0/1
1	10	.	T	TATATAC	.	.	.	GT	0/0
1	10	.	T	TATATATAC	.	.	.	GT	0/0
