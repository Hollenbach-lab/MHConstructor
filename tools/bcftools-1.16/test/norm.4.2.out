##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=249250621>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	10	.	T	C,TAC,TATAC,TATATAC,TATATATAC	.	.	.	GT	3/1
